pkgs/top-level/all-packages.nix

```diff
@@ -1184,6 +1184,10 @@ let
     python = python26;
   };
 
+  cudatoolkit7 = callPackage ../development/compilers/cudatoolkit/7.0.nix {
```

Comment:
I just copied the style from the generic ffmpeg calls, which also seem to call it twice.